locksmith/config/config.js

```diff
@@ -41,4 +45,9 @@ if (process.env.DATABASE_URL) {
   }
 }
 
+if (process.env.UNLOCK_ENV === 'prod' || process.env.UNLOCK_ENV === 'staging') {
+  config.services.wedlocks = 'https://wedlocks.unlock-protocol.com/.netlify/functions/handler'
```

Comment:
How do you stop someone else from spamming that specific endpoint? 

Afaik, there are no easy way to get static IPs on heroku so you can't rely on IP whitelist.

Reply:
For now we don't. Since we only send emails from pre-defined templates, the value for 3rd party folks is pretty little. That said we had a plan to "sign" the API requests. I think once that gets a probem we should absolutely do it!